CardinalKit-Example/CardinalKit/Library/CareKit/CKCareKitManager+Sample.swift

```diff
@@ -24,14 +25,26 @@ internal extension OCKStore {
         let group = DispatchGroup()
         for document in documents{
             group.enter()
-            CKSendHelper.getFromFirestore(authCollection:authCollection, collection: collection, identifier: document.documentID) {(document, error) in
-                do{
-                    guard let document = document,
-                          let payload = document.data(),
-                          let id = payload["id"] as? String else {
-                              group.leave()
-                        return
-                    }
+            var route = ""
+               if let authCollection = authCollection {
+                   route = "\(authCollection)\(collection)/\(document.documentID)"
+               }
+               else{
+                   guard  let nAuth = CKStudyUser.shared.authCollection else {
+                       return
+                   }
+                   route = "\(nAuth)\(collection)/\(document.documentID)"
+               }
+               CKApp.requestData(route: route, onCompletion: {
```

Comment:
Same here on authCollection. Should the route be defined within the library to avoid confusion for the developer?

Reply:
I agree, I think it should be defined in the library. However, there are some exceptions where the developer would need to be able to easily change the route - for example with a recent project RIT asked for a specific path to be used.